queries/git_config/injections.scm

```diff
@@ -6,3 +6,26 @@
   value: (string) @injection.content)
   (#any-of? @_name "cmd" "command" "textconv")
   (#set! injection.language "bash"))
+
+(section
+  (section_header
+    (section_name) @_section)
+  (variable
+    value: (string) @injection.content)
+  (#eq? @_section "alias")
+  (#lua-match? @injection.content "^!")
+  (#offset! @injection.content 0 1 0 0)
+  (#set! injection.language "bash"))
```

Comment:
Firstly, is there a documentation for this? 
Secondly, can this be anything else other than bash (I'll presume not because even windows have git-bash)

Reply:
I think this is good --  https://git-scm.com/docs/git-config#Documentation/git-config.txt-alias

> If the alias expansion is prefixed with an exclamation point, it will be treated as a shell command

Reply:
So from the doc, I assume the command can also be a powershell command. I'm somewhat against this because of that

Reply:
Ah, I see. I will close then. Maybe one day we can get a fancy directive that gets the current user shell and injects that one for things like this :joy: However, I think it is worth noting that `bash` is injected for other commands:
```scheme
((variable
  (name) @_name
  value: (string) @injection.content)
  (#any-of? @_name "cmd" "command" "textconv")
  (#set! injection.language "bash"))
```
From the docs it looks like these are also just run in the general shell. Should this be kept?

Reply:
:thinking: I guess if it's for consistency purposes then your PR should also be kept as well.

Reply:
Sounds good. I am curious how Github handles this, it seems to have some sort of injection for the alias/command things.